src/shared/helpers/helper.util.ts

```diff
@@ -84,86 +80,25 @@ export function isElrondChain(chain: string) {
 	return chain.includes('elrond')
 }
 
-/**
- * Formats chainId to its name
- */
-export function formatChainName(chainId: string) {
-	return (
-		EIP155_CHAINS[chainId as TEIP155Chain]?.name ??
-		COSMOS_MAINNET_CHAINS[chainId as TCosmosChain]?.name ??
-		SOLANA_CHAINS[chainId as TSolanaChain]?.name ??
-		NEAR_TEST_CHAINS[chainId as TNearChain]?.name ??
-		ELROND_CHAINS[chainId as TElrondChain]?.name ??
-		chainId
-	)
-}
-
 /**
  * Formats url to to remove protocol
  */
 export function formatUrl(url: string) {
 	return url.split('//')[1]
 }
 
-/**
- * Formats chainId to its name
- */
-export function getTicker(chain: string) {
-	switch (chain) {
-		case 'ethereum':
-		default:
-			return 'ETH'
-		case 'polygon':
-			return 'MATIC'
-		case 'bsc':
-			return 'BNB'
-		case 'arbitrum':
-			return 'ARB'
-		case 'optimism':
-			return 'OP'
-		case 'ethereum_goerli':
-			return 'GETH'
-		case 'optimism_goerli':
-			return 'OP'
-	}
-}
+export const getNetwork = (chain: string) => SUPPORTED_NETWORK[chain]
 
-export const getNamespace = (chain: string) => {
-	switch (chain) {
-		case 'ethereum':
-		default:
-			return 'eip155:1'
-		case 'polygon':
-			return 'eip155:137'
-		case 'bsc':
-			return 'eip155:56'
-		case 'optimism':
-			return 'eip155:10'
-		case 'arbitrum':
-			return 'eip155:42161'
-		case 'ethereum_goerli':
-			return 'eip155:5'
-		case 'optimism_goerli':
-			return 'eip155:420'
-	}
-}
+export const getTicker = (chain: string) => SUPPORTED_NETWORK[chain].ticker
+
+export const getDisplayName = (chain: string) =>
+	SUPPORTED_NETWORK[chain]?.displayName ?? chain
+export const getNamespace = (chain: string) =>
+	SUPPORTED_NETWORK[chain].namespace ?? chain
 
 export const getCurrencyByChainId = (chainId: string) => {
-	switch (chainId) {
-		case 'eip155:1':
-		default:
-			return 'ethereum'
-		case 'eip155:137':
-			return 'polygon'
-		case 'eip155:56':
-			return 'bsc'
-		case 'eip155:10':
-			return 'optimism'
-		case 'eip155:42161':
-			return 'arbitrum'
-		case 'eip155:5':
-			return 'ethereum_goerli'
-		case 'eip155:420':
-			return 'optimism_goerli'
-	}
+	const elem = Object.entries(SUPPORTED_NETWORK).find(
```

Comment:
You can use Object.values instead of Object.entries if we just need the networks

Reply:
So elem as to be ` [string, Network]`

Reply:
My bad x)